backend/src/utils/requestError.ts

```diff
@@ -12,6 +12,27 @@ export enum LogLevel {
     EMERGENCY = 600,
 }
 
+export const mapToWinstonLogLevel = (customLogLevel: LogLevel): string => {
```

Comment:
removing as not needed for time being. each error should have it's own log level already